lib/common_test/src/ct_ssh.erl

```diff
@@ -367,125 +392,188 @@ receive_response(SSH, ChannelId, Timeout) when is_integer(Timeout) ->
 %%%      <code>false</code> to wait for more data from the server.
 %%%      (Note that even if a fun is supplied, the function returns
 %%%      immediately if the server closes the channel).</p>
+%%% @end
+-spec receive_response(SSH, ChannelId, End, Timeout) ->
+                              {ok,Data} | {timeout,Data} | {error,Reason} when
+      SSH :: connection(),
+      ChannelId :: integer(),
+      End :: Fun | close | timeout,
+      Fun :: fun((ssh_connection:ssh_event_msg()) -> boolean()),
```

Comment:
Doc says: `data value in a received ssh_cm message (see ssh_connection(3)`
Which leads to `{ssh_cm, ssh_connection_ref(), ssh_event_msg()}` and I guess the data is `element(3)` of this tuple.
Then I could pick in one of these 3, but doc isn't explicit enough.
`ssh_event_msg() = data_events() | status_events() | terminal_events()`

What should I do?